.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: node_js
 node_js:
   - '9'
-  - '8'
-  - '7'
-  - '6'
+  - '8' 
+
+install:
+  - npm install
+  - npm install --only=dev
+
+script:
+  - npm run test
```

Comment:
Default build script in `.travis.yml` is `npm test`, so I think it would not need to add this part.

https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Default-Build-Script

Reply:
Oh, I didn't notice that, but don't you think write it exactly in `.travis.yml` make it more clear when maintaining this script ?

Reply:
Yep, I agree with you. Let @kpman decide whether to add it or not.

Reply:
The travisCI default script is `npm run test` and the project contains yarn.lock file.
It means travisCI will use yarn instead of npm install.

So I don't think this PR will be helpful.

If you want to contribute to this project, maybe you can add some useful config like cache node_modules and so on :)